integrations/jina/pyproject.toml

```diff
@@ -25,7 +25,7 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: CPython",
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
-dependencies = ["requests", "haystack-ai"]
+dependencies = ["requests>=2.25.0", "haystack-ai"]
```

Comment:
doesn't work with haystack 2.4 ?

Reply:
I don't remember... `requests>=2.25.0` is the lowest compatible version with `jina-haystack`